fly/eventstream/render.go

```diff
@@ -25,9 +26,13 @@ func Render(dst io.Writer, src eventstream.EventStream, options RenderOptions) i
 		if err != nil {
 			if err == io.EOF {
 				return exitStatus
+			} else if options.IgnoreEventParsingErrors && isEventParseError(err) {
+				dstImpl.SetTimestamp(0)
+				fmt.Fprintf(dstImpl, "failed to parse next event: %s\n", ui.ErroredColor.Sprint(err))
```

Comment:
Do we need to log this? Presumably if they're passing the flag, they've already encountered that there is an unknown event type/version. Also, it is called `--ignore-event-parsing-errors`, so I think makes sense to ignore them completely

Reply:
yea, thats fair 👍